app/src/test/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingAutomaticSignInViewHolderTest.kt

```diff
@@ -5,14 +5,12 @@
 package org.mozilla.fenix.home.sessioncontrol.viewholders.onboarding
```

Comment:
https://github.com/mozilla-mobile/fenix/pull/20819 
Nice to have a 2nd pair of eyes on the same file 😆